release-notes/fleet-elastic-agent/index.md

```diff
@@ -23,6 +23,17 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [fleet-elastic-agent-next-fixes]
 % *
 
+## 9.0.1 [fleet-elastic-agent-9.0.1-release-notes]
+
+### Features and enhancements [fleet-elastic-agent-9.0.1-features-enhancements]
+
+* Reuse shared integration policies when duplicating {{agent}} policies in {{fleet}} [#217872](https://github.com/elastic/kibana/pull/217872)
+* Update OTel components to v0.121.0 [#7686]({{agent-pull}}7686)
+* Add nopexporter to Elastic Distribution of OTel Collector (EDOT) Collector [#7788]({{agent-pull}}7788)
+* In {{agent}}, use `fullnameOverride` to set the fully qualified application names in the EDOT Kube-Stack Helm chart. [#7754]({{agent-pull}}7754) [#7754]({{agent-issue}}7754)
+
+### Fixes [fleet-elastic-agent-9.0.1-fixes]
+* In {{agent}}, fix Managed OTLP Helm config to use the current image repository [#7882]({{agent-pull}}7882)
```

Comment:
This also only applies the EDOT collector.

Reply:
Thanks. Changed to `In the EDOT Collector, fix Managed OTLP Helm config to use the current image repository`.